tests/integration/test_redirection_absolute.py

```diff
@@ -0,0 +1,49 @@
+"""Check REDIRECTIONS"""
+
+import io
+import os
+import shutil
+
+import pytest
+
+import nikola.plugins.command.init
+from nikola import __main__
+
+from .helper import add_post_without_text, append_config, cd
+from .test_empty_build import (  # NOQA
+    test_archive_exists,
+    test_avoid_double_slash_in_rss,
+    test_check_files,
+    test_check_links,
+    test_index_in_sitemap,
+)
+
+
+@pytest.fixture(scope="module")
```

Comment:
Please restructure this.

1. Merge all three `test_redirection_*.py` files into one (create all three redirects in one site)
2. Write a test to check that each of the three redirects works (file is created and has a meta refresh tag with the appropriate URL)

Reply:
I've merged all redirection tests into one file (`tests/integration/test_redirections.py`) and added the tests.
What do you say about it now?

Reply:
Much better.